future.go

```diff
@@ -0,0 +1,126 @@
+package tarantool
```

Comment:
It seems, that these changes are not related to the issue or related indirectly, maybe it is worth to move it in a separate commit.

Reply:
Good idea. Done.